Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -481,7 +481,22 @@ private void MeleeDamage(FPSWeapon weapon)
 
                 // TODO: Use correct damage based on weapon and swing type
                 // Just using fudge values during development
```

Comment:
I found in the manual for The Elder Scrolls: Arena some information.

Diagonal slash left: Pull the mouse sharply -5% to hit, +2 damage
down to the left
Diagonal slash right: Pull the mouse sharply -5% to hit, +2 damage
down to the right
Horizontal cut: Pull the mouse left or right - 0 -
Vertical chop: Pull the mouse downward -10% to hit, +4 damage
Thrust: Push the mouse upward +10% to hit, -4 damage

These are the same attack types as Daggerfall, and although there aren't exact numbers in the Daggerfall manual the descriptions match. This seems like a good starting point, and might even be the same thing.